### DISCUSSION TITLE
Continual Training in AutoGluon's Time Series Module

### DISCUSSION THREAD

**Discussion by usamabuttar** [autogluon/autogluon#4192] [Q&A] [answered]
Hey everyone, I have a question about AutoGluon's Time Series module. I'd like to know if it's designed for continual training. Here's the scenario: I receive daily updates for sales data, and my objective is to forecast sales for the following day. Do I need to initiate a fresh training session every day, essentially starting from scratch? Or does AutoGluon have the ability to smoothly continue training, improving the model with the most recent data?

**Comment by shchur** [comment, marked as answer]
Hi @usamabuttar, we do not support incremental training in time series for multiple reasons
- For deep learning based models, there is no clear best practice for incremental training. Practice followed by many is to retrain from scratch (or better, on a large trailing window) with each additional window, and this is often not a huge cost at the size of the models we use in practice.
- For tree-based models, incremental training is both not available in our implementations and unclear whether they are consistently effective.
- For the rest of our models, i.e., statistical models, "training" is anyway not really performed. These models are fit during inference on a trailing window. So incremental training is not defined.
